pom.xml

```diff
@@ -40,6 +40,7 @@
     <!-- ====================================================================== -->
     <modules>
         <module>parent</module>
+        <module>ui.frontend</module>
```

Comment:
Can we rename this module to af.renderer ?

Reply:
we can discuss a better name. its not a AF renderer technically.

Reply:
I had a discussion with @vdua on this